### PR TITLE
fix: Warpified component classes for Switch component

### PR DESCRIPTION
**Pull request by magnuh**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -127,18 +127,18 @@ export const card = {
...
-  switchThumb:
-    'absolute transition-gpu h-16 w-16 top-4 left-4 rounded-full transition-transform',
-  switchThumbSelected: 'f-switch-selected',
-  switchThumbNotDisabled: 'bg-white shadow',
-  switchThumbDisabled: 'bg-bluegray-300',
+  track: 'absolute h-full w-full rounded-full transition-colors',
+  trackActive: 'i-bg-$color-switch-track-background-active group-hover:i-bg-$color-switch-track-background-active-hover',
+  trackInactive: 'i-bg-$color-switch-track-background group-hover:i-bg-$color-switch-track-background-hover',
+  trackDisabled: 'i-bg-$color-switch-track-background-disabled',
+  handle: 'absolute transition-gpu h-16 w-16 top-4 left-4 rounded-full transition-transform',
```
Comment (AnnaRybkina) [inline index.js]: I think `transition-gpu` is depricated, it should be `transform-gpu` or `will-change-*`

Comment (magnuh) [reply index.js]: Hmm.... has that changed? Or was it plain wrong in Fabric? 😄

Comment (magnuh) [reply index.js]: Fixed

Comment (AnnaRybkina) [reply index.js]: We added this to the `preset-migrate` as it was removed from Drive(uno). So I assume it worked in TW

**Review by AnnaRybkina — APPROVED** [review]

**Pull request merged**